hivemq-edge/src/frontend/src/modules/Workspace/components/nodes/NodeBridge.tsx

```diff
@@ -35,7 +35,7 @@ const NodeBridge: FC<NodeProps<Bridge>> = ({ id, selected, data: bridge, draggin
       >
         {!showSkeleton && (
           <VStack>
-            {options.showTopics && <TopicsContainer topics={topics.remote} />}
+            {options.showTopics && <MappingBadge destinations={topics.remote.map((topic) => topic.topic)} />}
```

Comment:
suggestion (not tested, could be wrong):
```ts
{options.showTopics && <MappingBadge destinations={topics.remote.map(({ topic }) => topic)} />}
```

Same suggestion on line 50 (if it works here).

Reply:
I renamed the variable, it makes better sense to have `topics.remote.map((filter) => filter.topic`